Classes/Mapper/OpenStreetMapMapper.php

```diff
@@ -66,14 +66,15 @@ protected function getFormattedAddress(array $data)
         if (array_key_exists('address', $data)) {
             $data['address']['city'] = $this->getCityFromAddress($data['address']);
 
-            $formattedAddress = trim(sprintf(
+            $formattedAddress = sprintf(
                 '%s %s, %s %s, %s',
                 $data['address']['road'] ?? '',
                 $data['address']['house_number'] ?? '',
                 $data['address']['postcode'] ?? '',
                 $data['address']['city'] ?? '',
                 $data['address']['country'] ?? ''
-            ), ' ,\t\n\r\0\x0B');
+            );
+            trim(' ,\t\n\r\0\x0B');
```

Comment:
What are you doing here? Please move that `trim` below if-condition and apply it to `return $formattedAddress;`